crd-generator/src/main/java/io/strimzi/crdgenerator/CrdGenerator.java

```diff
@@ -305,9 +305,16 @@ public CrdGenerator(VersionRange<KubeVersion> targetKubeVersions, ApiVersion crd
         this.storageVersion = storageVersion;
         this.servedVersion = servedVersions;
         this.conversionStrategy = conversionStrategy;
+
+        //noinspection ReplaceNullCheck
```

Comment:
What's this?

Reply:
A hint to Intellij to stop suggesting a particular refactoring. It keeps suggesting replacing this:

```
        if (defaultValueHandler == null) {
            this.defaultValueHandler = new DefaultValueHandler(crdApiVersion, this::err);
        } else {
            this.defaultValueHandler = defaultValueHandler;
        }
```

With 

```
        this.defaultValueHandler = Objects.requireNonNullElseGet(defaultValueHandler, 
                () -> new DefaultValueHandler(crdApiVersion, this::err));
                
```

And highlights it for IDEA users as a warning, accordingly, so that comment turns it off for that statement.